src/.nuget/dir.targets

```diff
@@ -93,6 +93,49 @@
     </ItemGroup>
   </Target>
 
+  <!--
+    Creates listings of native files that the IL Linker should keep, and injects them
+    into the package build.
+    Happens after GetPackageDependencies. This is essentially just before GetPackageFiles, but
+    avoids creating these text files for ProjectReferences which have GetPackageFiles called
+    recursively.
+  -->
+  <Target Name="GetLinkerNativeIncludes" AfterTargets="GetPackageDependencies">
+    <ItemGroup>
+      <LinkerRuntimeFiles Include="@(NativeBinary)" Condition=" '%(NativeBinary.LinkerUsage)' == 'runtime' " />
+      <LinkerDiagnosticFiles Include="@(NativeBinary)" Condition=" '%(NativeBinary.LinkerUsage)' == 'diagnostic' " />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <HasLinkerRuntimeFiles Condition=" '@(LinkerRuntimeFiles->Count())' != '0' ">true</HasLinkerRuntimeFiles>
```

Comment:
We'll always have LinkerRuntimeFiles and LinkerDiagnosticFiles, right? Can we make this an assertion instead of a condition?

Reply:
The jit package doesn't have any diagnostic files.